app/jobs/start_sync_job.rb

```diff
@@ -1,12 +1,20 @@
 class StartSyncJob < ApplicationJob
   attr_accessor :sync, :part_size
 
-  def perform(sync_id, part_size = OrganizationExportJob::PART_SIZE)
+  PART_SIZE = OrganizationExportJob::PART_SIZE
```

Comment:
I can't think of a better way to do this, but it's def something I'm going to keep an eye out for.